extensions/blocks/podcast-player/style.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Frontend (site) styles for Podcast Episodes
+ */
+
+.wp-block-jetpack-podcast-player {
+    border: 1px solid;
+}
+
+.wp-block-jetpack-podcast-player audio {
+    display: none;
```

Comment:
This will make the audio element inaccessible to screenreaders. Is that what you want to do here? If not then consider a method to visually hide without doing so for screenreaders.

Reply:
I'm on the edge here. This audio element is never used and all player interactions are done through custom HTML components rendered by ME.js. The only case this player could be useful in any way is when you have JS disabled and that not a very common situation now and at the same time, the audio player can only play the very first episode. I think we might be better off hiding it completely and let users access those episode links directly.

Reply:
I'm actually thinking about removing the `<audio>` from the static markup and only leave the list, since it cannot be used fully anyway without JS

Reply:
Yep ok. In which case just leave the links in and allow folks to access those to play. Remove the `<audio>`.

Reply:
Done. The way it loads the audio is not amazing looking but it will also be replaced very soon using the data from parsed JSON. At least one element with data attribute `data-jetpack-podcast-audio` will always be present, otherwise we don't render the block markup and JS won't be run on it.